tutorials/maprapp/src/main/resources/META-INF/properties.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<configuration>
+  <!-- memory assigned to app master
+  <property>
+    <name>dt.attr.MASTER_MEMORY_MB</name>
+    <value>1024</value>
+  </property>
+  -->
+
```

Comment:
It will be helpful to add a note in README.md indicating which of these properties need to be modified for the user's environment.